concepts/get-started-transforming.md

```diff
@@ -0,0 +1,399 @@
+reusing the `map` operator, passing it a function to grab the appropriate
+property:
+
+[(Source Code)](https://github.com/ReactiveX/rxjs/blob/e17df333fec66ea3d79e3f70565064f757c3a4fe/src/internal/operators/pluck.ts#L48-L54)
+
+```ts
+export function pluck<T, R>(...properties: string[]): OperatorFunction<T, R> {
+  const length = properties.length;
+  if (length === 0) {
+    throw new Error('list of properties cannot be empty.');
+  }
+  return (source: Observable<T>) =>
+    map(plucker(properties, length))(source as any);
+}
+```
+
+Functionally, `map` and `pluck` will operate the same in these scenarios, I
+would suggest using whichever you feel most comfortable reading at a glance.
+
+Lastly, there may also be times where you **always** want to map to a single
+value, no matter the input. For these situations, you can use the `mapTo`
+operator.
+
+## Mapping to a constant value with `mapTo`
+
+![mapTo](https://drive.google.com/uc?export=view&id=1329klWDEvjgjh3JVJkzHoZDc7CpcLZvO)
+
+For situations where you find yourself always wanting to map to a specific
+value, one way you could handle it is by simply using `map` and ignoring the
+input:
+
+```js
+import { fromEvent } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+const click$ = fromEvent(document, 'click');
+
+keyup$
+  .pipe(map(() => 'You clicked!'))
+  // 'You clicked!', 'You clicked!'
+  .subscribe(console.log);
+```
+
+While this works, the wrapping function isn't necessary since we are ignoring
+the received value. For these scenarios you can replace `map` with `mapTo`, and
+simply provide the value you wish to return on all emissions:
+
+```js
+import { fromEvent } from 'rxjs';
+import { mapTo } from 'rxjs/operators';
+
+const click$ = fromEvent(document, 'click');
+
+click$
+  .pipe(mapTo('You clicked!'))
+  // 'You clicked!', 'You clicked!'
+  .subscribe(console.log);
+```
+
+Like `pluck`, `mapTo` provides no real benefit functionally over returning a
+constant value with `map`, but syntactically it may prove slightly easier to
+consume and read at a glance.
+
+## Conclusion
+
+In conclusion, `map` is a versatile operator which lets you transform a stream
+using a provided projection function. Whether it's mapping to a keycode, value
+updates from an input box, or reshaping an object, `map` will be one of the most
+used operators in your day-to-day RxJS toolbox. For scenarios where you just
+need to map to a single property, or always want to map to a constant value, you
+can also check out the [`pluck`](../operators/transformation/pluck.md) and
+[`mapTo`](../operators/transformation/mapto.md) helper operators.
+
+For a full list of transformation operators with examples, including operators
+which manage mapping to more complex values such as other observables, check out
+the [transformation operator section](../operators/transformation/README.md). We
+will explore these topics in detail in future posts!
```

Comment:
Maybe worth to mention https://www.learnrxjs.io/learn-rxjs/concepts/time-based-operators-comparison here? These are transform operators too :)

Reply:
Good suggestion, need to update my wording a bit to fit this in.